src/transaction/components/TransactionButton.tsx

```diff
@@ -31,9 +30,7 @@ export function TransactionButton({
   const { showCallsStatus } = useShowCallsStatus();
 
   const isInProgress =
-    statusWriteContract === 'pending' ||
-    statusWriteContracts === 'pending' ||
-    isLoading;
+    lifeCycleStatus.statusName === 'transactionPending' || isLoading;
```

Comment:
Not a fan of the name `transactionPending`, something to reflect more in the next few weeks.